scripts/travis/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Ensure that this is being run by Travis
+if [ "$TRAVIS" != "true" ] || [ "$USER" != "travis" ]; then
+  echo "This script should only be run by Travis CI."
+  exit 2
+fi
+
+# Output something every 9 minutes, otherwise Travis will abort after 10 minutes of no output
+while sleep 9m; do echo "[Ping] Keeping Travis job alive ($((SECONDS / 60)) minutes)"; done &
+WAITER_PID=$!
+
+# Skip tests if building a tag to prevent flaky releases
```

Comment:
Why do we need this? With the change to skip flaky tests this shouldn't be a problem, right?
Also, even if release job fails, we can retry again?

Reply:
It seems to me that it would be redundant. All PRs to master are tested, and all commits on master are tested. Given that release tags are always on master, we would essentially be testing the same code 3 times.

Reply:
Okay, makes sense.